src/gramps.js

```diff
@@ -65,19 +71,30 @@ export default function gramps(
     devSources,
     logger,
   });
-  const schema = getSchema({
-    sources,
-    logger,
-    options: apolloOptions.makeExecutableSchema,
-  });
 
-  if (enableMockData) {
-    addMockFunctions({
-      schema,
-      sources,
-      options: apolloOptions.addMockFunctionsToSchema,
-    });
-  }
+  const allSources = [rootSource, ...sources];
+  const schemas = allSources
+    .map(({ schema: typeDefs, resolvers, mocks }) => {
+      if (!typeDefs) {
+        return null;
+      }
+      const schema = makeExecutableSchema({
+        typeDefs,
+        resolvers,
+        ...apolloOptions.makeExecutableSchema,
+      });
+      if (enableMockData) {
+        addMockFunctionsToSchema({
+          schema,
+          mocks,
+          ...apolloOptions.addMockFunctionsToSchema,
+        });
+      }
+      return schema;
+    })
+    .filter(e => !!e);
```

Comment:
What is this catching? Can we make it more explicit?

Reply:
I actually had to do that to fix tests. If a data-source doesn't have a schema/typeDefs or resolvers then it can't be used to create a executable `schema`

Some "data sources" in the tests didn't have any typeDefs or resolvers